pkgs/development/python-modules/pygal/default.nix

```diff
@@ -22,7 +23,19 @@ buildPythonPackage rec {
     sha256 = "9204f05380b02a8a32f9bf99d310b51aa2a932cba5b369f7a4dc3705f0a4ce83";
   };
 
-  buildInputs = [ flask pyquery pytest ];
+  buildInputs = [
+    flask
+    pyquery
+
+    # Should be a check input, but upstream lists it under "setup_requires".
+    # https://github.com/Kozea/pygal/issues/430
```

Comment:
Should probably just do like https://github.com/NixOS/nixpkgs/blob/1b1ea35f74be12102f81eea9b72ccdb47bbb83ff/pkgs/development/python-modules/pyrsistent/default.nix#L24

Though I probably should have upstreamed that one.

Reply:
I don't think its worth maintaining a patch here. Is there any downside to just listing it in `buildInputs`? Its not propagated.